### PR TITLE
Feature/datasets version history

### PR DESCRIPTION
**Pull request by jamie-short**
Added the --version-history/-v flag to the get dataset command, to display the version history instead of the metadata for the given dataset.

To Do:
- Update tests for get.py

### CODE REVIEW

File: test/fixtures/dataset_fixtures.py [hunk 1]
```diff
@@ -262,6 +265,9 @@ def dataset_meta_mock(
         language (str, optional): Associated Language. Defaults to "en".
         standard (str, optional): Associated standards. Defaults to "ISO 9001".
         update (str, optional): Frequency updated. Defaults to "Annual".
+        title (str, optional): Associated Title. Defaults to "en".
+        dataset_id (str, optional): Dataset ID. Defaults to "ISO 9001".
+        version_id (str, optional): Dataset Version ID. Defaults to "Annual".
```
Comment (iroper-tessella) [inline test/fixtures/dataset_fixtures.py:270]: These defaults aren't correct.

**Review by iroper-tessella — CHANGES_REQUESTED** [review]
Could you add some documentation for the `--version-history` option for get dataset please?

File: dafni_cli/datasets/dataset_metadata.py [hunk 2]
```diff
@@ -149,6 +157,13 @@ def set_details_from_dict(self, dataset_dict: dict):
         self.standard = check_key_in_dict(dataset_dict, ["dct:conformsTo", "label"])
         self.update = check_key_in_dict(dataset_dict, ["dct:accrualPeriodicity"])
 
+        # Version History Fields
+        self.dataset_id = check_key_in_dict(
+            dataset_dict, ["version_history", "dataset_uuid"]
+        )
+        self.title = check_key_in_dict(dataset_dict, ["dct:title"])
+        self.version_id = version_id
```
[the remark this answers is not in the document]
Comment (jamie-short) [reply dafni_cli/datasets/dataset_metadata.py:165]: good point, hadn't noticed the extra key in the response, updated

File: test/fixtures/dataset_fixtures.py (hunk 1 above)
Comment (jamie-short) [reply test/fixtures/dataset_fixtures.py:270]: updated

File: test/fixtures/dataset_fixtures.py [hunk 3]
```diff
@@ -262,6 +265,9 @@ def dataset_meta_mock(
         language (str, optional): Associated Language. Defaults to "en".
         standard (str, optional): Associated standards. Defaults to "ISO 9001".
         update (str, optional): Frequency updated. Defaults to "Annual".
+        title (str, optional): Associated Title. Defaults to "en".
```
Comment (iroper-tessella) [inline test/fixtures/dataset_fixtures.py:268]: This one has default "Title".

**Review by iroper-tessella — CHANGES_REQUESTED** [review]
One tiny change needed. Then happy to be merged.

**Review by iroper-tessella — APPROVED** [review]

**Pull request merged**